backend/src/test/java/com/funeat/recipe/application/RecipeServiceTest.java

```diff
@@ -163,6 +177,98 @@ class create_실패_테스트 {
         }
     }
 
+    @Nested
+    class findRecipeByMember_성공_테스트 {
+
+        @Test
+        void 사용자가_작성한_꿀조합을_조회한다() {
+            // given
+            final var member1 = 멤버_멤버1_생성();
+            복수_멤버_저장(member1);
```

Comment:
```suggestion
            단일_멤버_저장(member1);
```
이거 생각해보니까 굳이 단일, 복수 나눠서 사용하지 않고, saveAll()로 통일해서 `멤버_저장`으로 해도 될 것 같네요

Reply:
이렇게 되면 모든 메소드를 다 수정해야될 것 같아서 이거는 수정사항에 달고 얘기해봐용!